src/Controller/CaseController.php

```diff
@@ -38,10 +42,60 @@ public function summary(CaseEntity $case): Response
     /**
      * @Route("/{id}/information", name="case_information", methods={"GET"})
      */
-    public function information(CaseEntity $case): Response
+    public function information(CaseEntity $case, ComplaintCategoryRepository $categoryRepository): Response
     {
+        /** @var ComplaintCategory $complaintCategory */
+        $complaintCategory = $categoryRepository->findOneBy([
+            'board' => $case->getBoard(),
+            'municipality' => $case->getMunicipality(),
+            'name' => $case->getCaseType(),
+        ]);
+
         return $this->render('case/information.html.twig', [
```

Comment:
Also the related template should be renamed `case/show.twig.html`

Reply:
Used renamed template `case/show.html.twig`.